Libraries/Text/TextStylePropTypes.js

```diff
@@ -49,7 +49,8 @@ const TextStylePropTypes = {
   textShadowRadius: ReactPropTypes.number,
   textShadowColor: ColorPropType,
   /**
-   * @platform ios
+   * Specifies the letter spacing (kerning). Defaults to 0.0 (no kerning) and is supported on iOS and 
```

Comment:
Nitpick: Kerning seems to be the wrong term per [Wikipedia](https://en.wikipedia.org/wiki/Letter-spacing).

> Letter-spacing should not be confused with kerning. [...] Kerning is a spacing adjustment of one or more specific pairs of adjacent characters [...]